src/executor/spec/executor/Linearization.spec.ts

```diff
@@ -17,13 +17,13 @@ test( 'Linearization of linear graph', () => {
     nodes: testNodes,
   }
 
-  const linearizedFlow = linearize( testDataflow )
+  const linearizedFlow = await linearize( testDataflow )
 
   expect( isLinearized( linearizedFlow ) ).toBe( true )
 
 } )
```

Comment:
Same thing as above. I don't think the linearize-function needs to be async.
Multiple lines in this file are affected, but I don't think every line needs it's own comment